content/en/blog/_posts/2020-08-21-Moving-Forward-From-Beta/index.md

```diff
@@ -61,17 +61,17 @@ mind.
 ## Avoiding permanent beta
 
 For Kubernetes REST APIs, when a new feature's API reaches beta, that starts a countdown.
-The beta-quality API now has **nine calendar months** to either:
+The beta-quality API now has **three releases** (about nine calendar months) to either:
 - reach GA, and deprecate the beta, or
 - have a new beta version (_and deprecate the previous beta_).
 
 To be clear, at this point **only REST APIs are affected**. For example, _APIListChunking_ is
 a beta feature but isn't itself a REST API. Right now there are no plans to automatically
 deprecate _APIListChunking_ nor any other features that aren't REST APIs.
 
-If a REST API reaches the end of that 9 month countdown, then the next Kubernetes release
+If a REST API reaches the end of that countdown, then the next Kubernetes release
```

Comment:
@saschagrunert , `countdown` seems vague.
`If a REST API reaches the end of that countdown,`
How about something like:
If a beta API has not graduated to GA after three Kubernetes releases, then the next Kubernetes release ...

Reply:
Sure, changed as suggested :+1: